ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -265,6 +266,7 @@ tests:
 - as: aws-ipi-f9-longduration-mco-all
   cron: 3 17 6,18,27 * *
   steps:
+    allow_skip_on_success: true
```

Comment:
@rioliu-rh if it is true, post steps are not executed when pre and test step are successful.

but we need to execute post to release resource, so please do not set it as true.

Reply:
maybe it is ok if there is no `optional_on_success: true` in step of post for removing resource.

Reply:
Example working [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-fips-f1/1770964040256851968),
```
......
INFO[2024-03-22T02:22:58Z] Step aws-ipi-fips-f1-openshift-e2e-test-qe-report succeeded after 8s. 
INFO[2024-03-22T02:22:58Z] Step phase test succeeded after 1h39m30s.    
INFO[2024-03-22T02:22:58Z] Running multi-stage phase post               
INFO[2024-03-22T02:22:58Z] Skipping optional step aws-ipi-fips-f1-gather-network 
INFO[2024-03-22T02:22:58Z] Skipping optional step aws-ipi-fips-f1-gather-core-dump 
INFO[2024-03-22T02:22:58Z] Skipping optional step aws-ipi-fips-f1-gather-aws-console 
INFO[2024-03-22T02:22:58Z] Skipping optional step aws-ipi-fips-f1-gather-must-gather 
INFO[2024-03-22T02:22:58Z] Skipping optional step aws-ipi-fips-f1-gather-extra 
INFO[2024-03-22T02:22:58Z] Skipping optional step aws-ipi-fips-f1-gather-audit-logs 
INFO[2024-03-22T02:22:58Z] Running step aws-ipi-fips-f1-ipi-deprovision-deprovision. 
INFO[2024-03-22T02:30:32Z] Step aws-ipi-fips-f1-ipi-deprovision-deprovision succeeded after 7m33s. 
INFO[2024-03-22T02:30:32Z] Step phase post succeeded after 7m33s. 
```

Reply:
Thanks for info.

Reply:
@kuiwang02 which post step will be skipped, as I know only the optional (has `optional_on_success: true`) step will be skipped when this attribute is true.

Reply:
ref: https://docs.ci.openshift.org/docs/architecture/step-registry/#skipping-post-steps-on-success